test/unit/repos/customer-repo-test.js

```diff
@@ -48,4 +48,36 @@ describe('CustomerRepo', function () {
       assert.deepEqual(result, customers);
     });
   });
+
+  describe('findByPk', function () {
+    it('calls findByPk on Customer model', async function () {
+      const fake = sinon.fake();
+      sinon.replace(Customer, 'findByPk', fake);
+      const id = 1;
```

Comment:
we could use `const { id } = customer` everywhere it is required to avoid hardcoding a magic number.